### PR TITLE
Separate computer_vision.py into its own test of enormous size.

### PR DESCRIPTION
**Pull request by gjoliver**
## Why are these changes needed?

This test is too long.
Separate it into a different test.

## Related issue number

Issue #35551
Issue #35692

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [*] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Comment by gjoliver** [comment]
looks good. the separated test passed:
Screen Shot 2023-05-22 at 2 47 06 PM

File: doc/BUILD [hunk 1]
```diff
@@ -183,13 +183,23 @@ py_test_run_all_subdirectory(
...
     extra_srcs = [],
     tags = ["exclusive", "team:ml"],
 )
 
+py_test(
+    name = "computer_vision_test",
+    main = "source/ray-air/doc_code/computer_vision.py",
+    srcs = ["source/ray-air/doc_code/computer_vision.py"],
+    tags = ["exclusive", "team:ml"],
+    size = "enormous",
```
Comment (matthewdeng) [inline doc/BUILD:200]: Should this be `large`? Since it finishes in 566 < 900 secs.

```suggestion
    size = "large",
```

Comment (gjoliver) [reply doc/BUILD:200]: apparently it runs faster once I separated it out.
it was timing out consistently at 900s.

Comment (richardliaw) [reply doc/BUILD:200]: I thought we were supposed to avoid using the 'enormous' tag?

Comment (gjoliver) [reply doc/BUILD:200]: 😕 
so the options are splitting the actual script into 2.
or specify a different more specific limit?
do you have a preference?

**Review by richardliaw — APPROVED** [review]
approve as hotfix to unblock release.

**Pull request merged**